beacon-chain/cache/committee.go

```diff
@@ -18,9 +18,10 @@ var (
 	// a Committee struct.
 	ErrNotCommittee = errors.New("object is not a committee struct")
 
-	// maxShuffledIndicesSize defines the max number of shuffled indices list can cache.
-	// 3 for previous, current epoch and next epoch.
-	maxShuffledIndicesSize = 3
+	// maxCommitteeSize defines the max number of shuffled committees one can cache
+	// on per epoch basis.
+	// Due to reorgs, it's good to keep the old cache around for quickly switch over.
+	maxCommitteeSize = 10
```

Comment:
This name is a little vague, this is tracking how many committees? Not the size of the committees, right?